TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md

```diff
@@ -0,0 +1,119 @@
+* Ben Sternthal (BS) created meeting minutes based on the publically available recording
+* DM reviewed the agenda
+
+### Roll Call / Attendance
+
+A quorum was not required.
+
+#### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* Arm - Nathaniel McCallum+  / Michael Lu
+* Google - Cfir Cohen +  / Catalin Sandu 
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair) + / Simon Johnson
+* Meta - Eric Northup / Shankaran Gnanashanmugam
+* Microsoft - Alec Fernandez +
+* Red Hat - Lily Sturmann  / Yash Mankad 
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Bokdeuk Jeong (Samsung)
+* Chris Ramming (Broadcom)
+* Drasko Draskovic (Ultraviolet)
+* Henk Birkholz (Fraunhofer Institute)
+* Kanth Ghatraju (Oracle) 
+* Marc Meunier (Arm)
+* Mark Novak (JP Morgan Chase)
+* Matthieu Legré (Cysec)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Steveb Walli (Microsoft)
+* Sung Lee (Broadcom)
+* Thomas Fossati (Linaro)
+* Wenhui Zhang (Bytedance/TikTok)
+* Mike Bursell (CCC)
+* Sal Kimmich (LF)
+
+## Welcome New Community Members
+
+No new community members were introduced
+
+## Meeting Minute Approval & Old Business
+
+* Meeting minutes will be approved via Github.
+* Dan Middleton (DM) provided an update on Kernel tech talk
+* Dan Middleton (DM) and Sal Kimmich (SK) provided an update on finding Asia-friendly meeting times
+
+## New Business
+
+* DM stated that a new project lead mailing list has been created 
+* DM reviewed the RFI responses and due dates for each
+
+### RFI Responses
+
+DM reviewed the list of RFIs and their respective due dates and stated that there is an active email list [thread](https://lists.confidentialcomputing.io/g/tac/message/1254 ):
+ 
+* Due January 15: NIST SP 1800-28 (Data Confidentiality: Identifying and Protecting Assets Against Data Breaches)
+* Due January 25: NIST SP 800-226 (Guidelines for Evaluating Differential Privacy Guarantees)
+* Due February 2: Safe, Secure, and Trustworthy Development and Use of AI
+
+
+
+Mike Bursell (MB) reviewed comments and changes on NIST SP 800-226 and the content that he has rewritten. Discussion ensued. 
+
+## 2024 TAC Priorities
+
+DM presented the overall objective of the CCC TAC. Discussion ensued. DM then went on to review the 2024 objectives and key results. Additional notes were taken in the OKR Google doc.
+
+## New Project Onboarding
+
+DM reviewed the new project pipeline. DM went on to review the list of projects and which projects are missing mentors. Chris Ramming (CR) asked about how to best leverage CCC/LF resources including managing meetings. DM showed example documents. Action Item: It was agreed that there should be a project benefits section on the website. 
```

Comment:
Did we suggest an owner for this action?

Reply:
I did not catch one in the meeting recording. I would suggest Riaan.